rfc-139-enable-http3.md

```diff
@@ -0,0 +1,135 @@
+
+The aim of this transition to HTTP/3 is to allow an ever more diverse level of user connection quality to access GOV.UK, essentially maximising access for all users, no matter what connection they use. By focusing on the long-tail of our users with poor connections, we can effectively shorten it and in doing so make the internet "good enough" to use in more situations. This is all without effecting performance for users already on a good internet connection.
+
+### Browser support
+The support for HTTP/3 + QUIC in browsers currently isn't accurate on [Caniuse](https://caniuse.com/?search=http3). A PR to rectify this can be [seen here](https://github.com/Fyrd/caniuse/pull/5836). The browsers support for HTTP/3 at the time of writing is:
+
+- Chrome - Enabled 
+- Edge (chromium version) - Enabled
+- Firefox - Enabled in the next version (88)
+- Safari 14 - Disabled (currently behind a feature flag)
+
+Looking at the March 2021 analytics, approximately 55% of GOV.UK users are using a browser that can now take advantage of HTTP/3 + QUIC. Information about HTTP/3 and QUIC usage in Chrome can be [found here](https://blog.chromium.org/2020/10/chrome-is-deploying-http3-and-ietf-quic.html).
+
+## Usage on the web
+
+Although the HTTP/3 and QUIC specifications are still in draft, they are both used in production on large sites with billions of daily page views:
+
+### YouTube
+In the image below we see Chrome 89 (latest stable) connecting to YouTube and using `h3-Q050`. This is HTTP/3 with the latest version of Google QUIC (`Q050`).
+
+![YouTube using HTTP/3 + QUIC as the transport protocol.](rfc-139/youtube-h3.png)
+
+### Google Search
+In the image below we see Chrome 89 (latest stable) connecting to Google Search and using `h3-Q050`. This is HTTP/3 with the latest version of Google QUIC (`Q050`).
+![Google Search using HTTP/3 and QUIC.](rfc-139/google-search-h3.png)
+
+### Facebook
+Facebook are using HTTP/3 and QUIC in production, but are using the IETF version of QUIC (`h3-29`). Below we see Firefox 88 connecting via HTTP/3:
+![Facebook servers are also HTTP/3 and QUIC enabled.](rfc-139/fb-h3.png)
+
+## Other advantages to HTTP/3 + QUIC
+
+I've only covered a single advantage related to packet loss with web performance above, but HTTP/3 + QUIC comes with a number of other advantages:
+
+- Improvement in connection startup times via TLS v1.3 + [0-RTT](https://blog.cloudflare.com/introducing-0-rtt/)
+- Better security via end-to-end encryption at the transport layer
+- Establish an ecosystem that is easier to evolve from our current TCP usage
+
+Eventually our users will also be able to take advantage of [Connection Migration](https://tools.ietf.org/id/draft-tan-quic-connection-migration-00.html), where QUIC end-points will be able to migrate connections to different IP addresses and network paths at will. Although realistically, this is likely a number of years away.
```

Comment:
Could you explain what benefits that would bring? I know it's just an off the cuff comment, but I don't understand it, and it sounds interesting! I skimmed through the linked document but it talked more about the technical protocol rather than the benefits of the feature.

Reply:
Good idea, yes I will add a few bullet points to highlight this.

Reply:
Added info about connection migration and a link to a demo of it in action.